source/extensions/extensions_metadata.yaml

```diff
@@ -0,0 +1,579 @@
+envoy.access_loggers.file:
+  categories:
+  - envoy.access_loggers
+  security_posture: robust_to_untrusted_downstream
+envoy.access_loggers.http_grpc:
+  categories:
+  - envoy.access_loggers
+  security_posture: robust_to_untrusted_downstream
```

Comment:
I think I prefer to see `status` being explicitly called out as `stable` rather than defaulted; this reflects an opionionated stance; folks declaring extensions must state it upfront.

Reply:
ok - this just reflects the default that was being set in the macro - but i agree - we shouldnt allow a default here and people should declare its status